### PR TITLE
Separate apps for the scorecard and the api

### PR DESCRIPTION
**Pull request by longhotsummer**
Move the scorecard app into its own Django app, and use the sites framework to differentiate between them.

### CODE REVIEW

**Comment by longhotsummer** [comment]
@xybrnet @jbothma

**Comment by xybrnet** [comment]
Should profiles.py also move into scorecard?

**Comment by longhotsummer** [comment]
Updated to move profiles.py and geo.py. Thanks @xybrnet

**Comment by xybrnet** [comment]
👍

File: municipal_finance/settings.py [hunk 1]
```diff
@@ -50,6 +52,11 @@
     'corsheaders',
 )
 
+# Sites
+if DEBUG:
+    SITE_ID = 2  # Scorecard
```
Comment (jbothma) [inline municipal_finance/settings.py]: where's this used?

Comment (longhotsummer) [inline municipal_finance/settings.py]: In production, the sites framework uses the domain of the incoming request to work out what site is active. In development, we don't have a domain and this setting tells Django which site to use. The ID is the id of the corresponding Site object in the database.

What, random integers in settings files aren't your thing? :)

File: municipal_finance/migrations/0001_initial.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.5 on 2016-04-25 11:05
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.contrib.sites.models import Site
+
+
+def create_sites(apps, schema_editor):
```
Comment (jbothma) [inline municipal_finance/migrations/0001_initial.py]: did you write this manually or was this generated from something you did on the shell? I want to add municipal-finance.code4sa.org to either serve the API site or redirect to data.municipalmoney.org.za

**Pull request merged**